UnrealThirdPersonMP/README.md

```diff
@@ -0,0 +1,96 @@
+# PlayFab Multiplayer Unreal Game Server Sample
+
+## Prerequisites
+- Download Unreal Engine Source Build by following [these instructions](https://docs.unrealengine.com/4.26/en-US/ProgrammingAndScripting/ProgrammingWithCPP/DownloadingSourceCode/) from the Unreal website
+- Download Unreal Engine (this was tested on 4.26.2)
+- Download the [Unreal PlayfabGSDK Plugin folder](https://github.com/PlayFab/gsdk/tree/master/UnrealPlugin)
+- Download Visual Studio (the [community version is free](https://visualstudio.microsoft.com/vs/community/))
+- When running locally you must have a copy of [Local Multiplayer Agent](https://github.com/PlayFab/MpsAgent/releases)
+- A developer.playfab.com account (which you can create for free)
+- Clone the [MpsSamples repository](https://github.com/PlayFab/MpsSamples)
+	- In order to use it, you need to have installed [.NET Core](https://dotnet.microsoft.com/download/dotnet-core)
+
+## Getting Started
+
+#### Create a multiplayer game
+First, you need to start by creating a multiplayer game with a dedicated server! Thankfully, Unreal has a tutorial to help you 
+make a [third person multiplayer game from scratch](https://docs.unrealengine.com/4.26/en-US/InteractiveExperiences/Networking/QuickStart/).
+
+#### Setting up a dedicated server
+
+Once you have created your game, you can set up a dedicated server. A dedicated server is one that is dedicated to only hosting the game 
+and no local players play on the machine because everyone connected is a client. You can do this by following [these instructions]
+(https://docs.unrealengine.com/4.26/en-US/InteractiveExperiences/Networking/HowTo/DedicatedServers/) from Unreal's documentation.
+
+#### Integrate the Playfab GSDK plugin
+Once you've followed this tutorial and created your third person multiplayer game, then you can integrate the Unreal Playfab GSDK Plugin
+[using these instructions](https://github.com/PlayFab/gsdk/blob/master/UnrealPlugin/README.md).
+
+## How to use the packaged server and client
+
+Now that you have a packaged game server, you can also package your game client. Open your Unreal Engine Editor and go to
+
+To get the game client, click on File -> Package Project -> Build Target -> [game name] and then go to File -> Package Project -> Windows (64-bit), and choose
+what folder you want this file to be saved to. 
+
+# INSERT IMAGE HERE!
+
+The packaged game client will be in a folder called "WindowsNoEditor".
+
+
+### Testing Locally using LocalMultiplayerAgent
+
+Since we can only use container mode with the actual MPSAllocatorSample, the following instructions will be for container mode. Because of this, we first need to ensure that our zipped game server has all required system DLLs that are not included in the container image. 
```

Comment:
MPSAllocatorSample doesn't require container mode

Reply:
plus, this section talks about LocalMultiplayerAgent, why is MpsAllocatorSample mentioned?